src/System.Text.Json/src/Configurations.props

```diff
@@ -1,12 +1,10 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
+      netstandard;
       netcoreapp-Unix;
```

Comment:
Action item for @ahsonkhan: remove thes OS specific configurations and remove the runtime reference from this project.

Reply:
> remove thes OS specific configurations

I get why we should remove the runtime reference, but why do we **need** to remove the OS specific configs?

Reply:
Also, regarding runtime reference, you already removed it when we in-boxed S.R.CS.Unsafe, here: https://github.com/dotnet/corefx/pull/35526/files#diff-05e80536a428043aa968323767b874c0L78

Reply:
>  you already removed it 

doh, good call

> but why do we need to remove the OS specific configs

I suspect they were only there because you were compiling against runtime, having them uneccessarily complicates & bloats the package by placing the implementations in runtime folders, that are identical.  It's better to have a single RID-less configuration.

@joperezr since this is a simple cleanup do you mind doing it in this PR?

Reply:
I'll take care of it